app/javascript/dashboard/components/widgets/WootWriter/Editor.vue

```diff
@@ -162,6 +163,22 @@ export default {
     isPrivate() {
       this.reloadState();
     },
+
+    updateSelectionWith() {
```

Comment:
```suggestion
    updateSelectionWith(newText, oldTex) {
```

And add a check if they are not same too

Reply:
@nithindavid Doubt what in the case of inserting the same multiple emojis?

Reply:
Done https://github.com/chatwoot/chatwoot/pull/5865/commits/9440b5b6f3623ebb9cf909802b58f0dd43d49440